### PR TITLE
Add github action to lintern the code

### LINKED ISSUE #6: ci: setup basic github actions workflow
**Description**

The goal is to setup a simple github actions workflow to catch build and linting errors.

### PR DESCRIPTION
**Pull request by alberturria**
**Description**
This PR adds a new GH Action workflow in charge of both, compile the code, and run the lintern.

In this run we can see how the action is executed: https://github.com/alberturria/vscode-juve/actions/runs/3169432936/jobs/5161341398

Captura

closes #6

### CODE REVIEW

File: .github/workflows/lintern.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Lintern
+on: [push]
```
Comment (vince-fugnitto) [inline .github/workflows/lintern.yml]: @alberturria do you mind updating so we can also run the CI on `pull_request`?

File: .github/workflows/lintern.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: Lintern
+on: [push]
+jobs:
+  lintern:
```
Comment (vince-fugnitto) [inline .github/workflows/lintern.yml]: @alberturria what do we mean by `lintern`?

File: .github/workflows/lintern.yml (hunk 1 above)
Comment (alberturria) [reply .github/workflows/lintern.yml]: Sure, on it! 🏃

File: .github/workflows/lintern.yml (hunk 2 above)
Comment (alberturria) [reply .github/workflows/lintern.yml]: Woopsie! I thought that was a word. I was referring to the fact of `linting` the code. (Find stylistic errors, in our case, according to the rules defined in the `eslintrc.json` file).

Are you OK with changing the name of the file, action, and job to `Lint`, or do you have any other suggestion? 🙂

Comment (vince-fugnitto) [reply .github/workflows/lintern.yml]: No problem, and I think either `build`, `lint` or both would be fine 👍

**Review by vince-fugnitto — APPROVED** [review]
LGTM thanks 👍

**Pull request merged**